cypress/e2e/orangeHRM/scheduleCandidateprocess.cy.ts

```diff
@@ -0,0 +1,55 @@
+import addCandidate from "cypress/support/addCandidateHelper";
+import addCandidatePage from "cypress/support/pageObjects/mainPages/addCandidatePage";
+import loginPage from "cypress/support/pageObjects/mainPages/loginPage";
+import scheduleInterviewPage from "cypress/support/pageObjects/mainPages/scheduleInterviewPage";
+
+const myAddCandidate: addCandidate = new addCandidate();
+const myLoginPage: loginPage = new loginPage();
+const myAddCandidatePage: addCandidatePage = new addCandidatePage();
+const myScheduleInterviewPage: scheduleInterviewPage = new scheduleInterviewPage();
+describe('schedule candidate', () => {
+
+    beforeEach(() => {
+        // visiting orangeHRM website
+        cy.visit("web/index.php/auth/login");
+        // login
+        myLoginPage.login("Admin", "admin123");
+
+        // define a fixture contains candidate info
+        cy.fixture('candidateInfo').as('candidateInfo');
+    })
+
+
+    it('test', () => {
+        // bring data from fixture
+        cy.get('@candidateInfo').then((candidateData) => {
+            // add candidate via API and use the response then
+            myAddCandidate.addCandidateViaAPI(candidateData.data).then((res) => {
+                let id = res.data.id;
+                
+                // change candidate status to shortlisted using API
+                cy.api({
+                    method: 'PUT',
+                    url: `https://opensource-demo.orangehrmlive.com/web/index.php/api/v2/recruitment/candidates/${id}/shortlist`
+                })
```

Comment:
this also needs to be in a POM, I suggest it to be in a dataUtils file that contains dealing with any request for the backend